testsuite/tests/kuadrant/reconciliation/conftest.py

```diff
@@ -3,6 +3,20 @@
 import pytest
 
 
+@pytest.fixture(scope="module")
+def authorization(authorization):
+    """Add RHSSO identity"""
+    authorization.identity.anonymous("anonymous")
```

Comment:
Is this rhsso identity?

Reply:
It is not, it is anonymous identity, meaning anyone can access it

Reply:
above is incorrect docstring then